### PR TITLE
Tidy update

### PR DESCRIPTION
**Pull request by Shellyber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/35969

## Description
Updated Content installation playbook
Updated Tidy integration

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Shellyber** [comment]
Regarding the `pack_metadata` should it stay the same?

File: Packs/Tidy/Playbooks/playbook-Content_developer_setup.yml [hunk 1]
```diff
@@ -1,973 +1,959 @@
...
-  playbookInputQuery:
-  required: true
-  value:
-    simple: ${incident.tidygithubtoken}
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
 name: Content developer setup
-outputs: []
+description: 6.0.0
```
Comment (Itay4) [inline Packs/Tidy/Playbooks/playbook-Content_developer_setup.yml:6]: lets improve the description here

**Review by Itay4 — APPROVED** [review]
looks good

**Comment by Shellyber** [comment]
@Itay4 Please review again the description change and the pack metadata

**Review by Itay4 — APPROVED** [review]
looks good

**Comment by Shellyber** [comment]
@roysagi Please review the rn

**Pull request merged**